CHANGELOG.md

```diff
@@ -4,8 +4,18 @@ Change Log
 ## Next Version
 - Various bug fixes.
 
-## Version 1.4.0
+## Version 1.5.0
+- Added Golang stack parsing.
+- Added Sanitizer options minimization.
+- Added HELP_FORMAT for custom reproduce instructions.
```

Comment:
nit: IMO it's not obvious what this means

Reply:
Done!